MetaMask/metamask-extension
# E2e folder structure

## **Description**



1. Move the NFT, swaps and metrics tests into the main tests folder
2. Simplifies the logic to run all tests

## **Related issues**

Fixes: #21583 

## **Manual testing steps**

1. Ensure [metrics tests](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/61150/workflows/e628d566-cf84-409b-a5ac-07f687c9a602/jobs/1915817/parallel-runs/0/steps/0-106?invite=true#step-106-2186_74) are run 
2. Ensure [nft tests](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/61150/workflows/e628d566-cf84-409b-a5ac-07f687c9a602/jobs/1915817/parallel-runs/0/steps/0-106?invite=true#step-106-2794_65) are run 
3. Ensure [swaps tests](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/61150/workflows/e628d566-cf84-409b-a5ac-07f687c9a602/jobs/1915817/parallel-runs/0/steps/0-106?invite=true#step-106-4910_60) are run 
 

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
